### PR TITLE
[BREAKING] feat: stop supporting node <= 8

### PR DESCRIPTION
**Pull request by gxcsoccer**

### CODE REVIEW

**Comment by fengmk2** [comment]
@gxcsoccer please change your commit author

image

File: lib/byte.js [hunk 1]
```diff
@@ -147,18 +147,18 @@ function charCode(c) {
...
-  var handles = numbers[type];
-  var size = handles.size;
+Object.keys(numbers).forEach(function(type) {
+  const putMethod = 'put' + type;
+  const getMethod = 'get' + type;
+  const handles = numbers[type];
+  const size = handles.size;
 
-  ByteBuffer.prototype[putMethod] = function (index, value) {
+  ByteBuffer.prototype[putMethod] = function(index, value) {
```
Comment (fengmk2) [inline lib/byte.js]: use `class` style instead of prototype?

**Review by fengmk2 — APPROVED** [review]

**Comment by fengmk2** [comment]
need to fixed package.json too https://github.com/node-modules/byte/blob/master/package.json#L55

**Review by fengmk2 — CHANGES_REQUESTED** [review]
ignore me

**Review by fengmk2 — APPROVED** [review]

**Comment by gxcsoccer** [comment]
2.0.0

**Pull request merged**